### ISSUE TITLE
Not working on mac os 10.13

### ISSUE THREAD

**Issue by eggcaker** [nEXT-Browser/nEXT#37]
Just download the latest version dmg file, but when press `S-h v` or `S-h s` nothing happens, if I try to custom the key follow guide page : 

```
(define-key *global-map* (kbd "S-M-h v")
  (:input-complete *minibuffer* variable-inspect variable-complete :setup #'load-package-globals))

``` 
the nEXT not start, some error message say, cannot find global-map
did I miss some thing ?

**Comment by jmercouris** [comment]
Hi, firstly, I'm sorry for the issues! I know exactly what is happening. The inspect-variable feature is not part of the DMG available for download. The DMG is only available as version 0.05. Therefore, the DMG that you can download does not yet include this feature and will be released on 0.06.

If you look here: https://github.com/neXT-Browser/nEXT/tree/master/next/#release-timeline

You can see exactly which features are available in which version. If you'd like, you can compile the application yourself or wait for the next release. Once again, sorry!

**Comment by eggcaker** [comment]
Good to know ， I'll try to compile the dmg if I can. Close for how. Thanks!
